src/Models/WpComposite.php

```diff
@@ -164,6 +165,13 @@ public static function on_save_slug_change($postId, WP_Post $post)
         add_action('save_post', [__CLASS__, 'on_save_slug_change'], 5, 2);
     }
 
+    public static function addedTermRelationship(int $postID, int $termID, string $taxonomy)
+    {
+        if (($taxonomy === 'category' && $post = get_post($postID)) && $post instanceof WP_Post) {
```

Comment:
Maybe also check that the post_type is contenthub_composite
```suggestion
        if (($taxonomy === 'category' && $post = get_post($postID)) && $post instanceof WP_Post && $post->type == static::POST_TYPE) {
```